math/src/field/f64/tests.rs

```diff
@@ -271,6 +271,16 @@ fn quad_mul() {
     assert_eq!(expected, a * b);
 }
 
+#[test]
+fn quad_mul_base() {
+    let a = <QuadExtension<BaseElement>>::new(rand_value(), rand_value());
+    let b0 = rand_value();
+    let b = <QuadExtension<BaseElement>>::new(b0, BaseElement::ZERO);
+
+    let expected = a * b;
+    assert_eq!(expected, a.mul_base(b0));
```

Comment:
Reasonable test.

This is great.

Someday we should chat about more robust multiplication testing.

(a + random1) * (b + random2) is even better (for the cases where you have not found an expected result by hand).